src/components/Tooltip/PTooltip.vue

```diff
@@ -23,32 +21,40 @@
 
 <script setup lang="ts">
   import { type TooltipRootProps, type TooltipProviderProps, type TooltipContentProps } from 'radix-vue'
-  import { defineProps, computed } from 'vue'
+  import { computed } from 'vue'
   import TooltipContent from '@/components/Tooltip/PTooltipContent.vue'
   import TooltipProvider from '@/components/Tooltip/PTooltipProvider.vue'
   import TooltipRoot from '@/components/Tooltip/PTooltipRoot.vue'
   import TooltipTrigger from '@/components/Tooltip/PTooltipTrigger.vue'
 
-
-  // eslint-disable-next-line vue/no-unused-properties
-  const props = defineProps<TooltipRootProps & TooltipProviderProps & TooltipContentProps & { text?: string }>()
-
-
-  const delegatedRootProps = computed(() => {
-    // eslint-disable-next-line id-length, no-unused-vars
+  const props = withDefaults(
+    defineProps<TooltipRootProps & TooltipProviderProps & Omit<TooltipContentProps, 'asChild' | 'as'> & { text?: string }>(),
+    {
+      text: undefined,
+      // Mimicking radix-vue's TooltipRoot props. These need to be undefined rather than Vue's default behavior of
+      // coercing boolean props to false so that the default values are respected when forwarded.
+      defaultOpen: false,
+      open: undefined,
+      delayDuration: undefined,
+      disableHoverableContent: undefined,
+      disableClosingTrigger: undefined,
+      disabled: undefined,
+      ignoreNonKeyboardFocus: undefined,
```

Comment:
When I just removed the colons in order to fix the `v-bind` usages, the tooltip broke. this fixes it.